src/components/atoms/ErrorMessage/index.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+import { errorMessage } from "../../../assets/styles/colors";
+
+const ErrorMessage = styled.span`
+  display: flex;
+  font-size: 1.4rem;
+  color: ${errorMessage};
```

Comment:
I am not a fan of this name for a colour, maybe sth about `alert`? or simply `error`, so would be just skipping message. ;)